enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -0,0 +1,490 @@
+
+We also performed some back-of-the envelope calculations in order to illustrate the added cost of functional sharding in terms of actual money. A `c5.xlarge` on-demand instance in AWS (a compute optimized instance with 4 CPUs and 8GB RAM) costs 140$ per month or 18$ per GB per month.
+When using reserved capacity, the cost goes down to 11$ per GB per month for the same instance with a 1 year reservation.
+
+Our conclusion based on these factors is as follows: while sharing a Prometheus instance does come with better resource utilization, we do not believe it justifies the reduced resiliency that it also brings along.
+The resiliency of such a shared instance would be further reduced as more managed services are installed in a cluster. In addition, a shared Prometheus instance is often more volatile regarding resource utilization, which leads to users over provisioning resources to prevent OOMs by misconfiguration or complex querying patterns.
+Finally, even if at one point functional sharding does end up being a problem from a resource usage perspective, the monitoring operator will nevertheless support deploying multi-namespace monitoring stacks. This will effectively support the pattern of sharing a prometheus instance between managed services.
+
+### Migration
+
+There are multiple monitoring solutions that Addons currently use:
+
+* UWM and AppSRE has their monitoring stack for SaaS services, which they plan to migrate to UWM
+  * MSO is planned to have feature parity with UWM with the exception of global view with Platform Monitoring. We will initially use federation to close this gap (See )
+* Platform Monitoring
+  * The PM stack cannot be used for user/managed service monitoring anyway. Therefore no migration is predicted so far.
+* Observability Operator
+  * This is great work done by the Kafka team to get unblocked and as we see it has feature parity to UWM with addition of Dashboarding and Promtail (logging aggregation). MSO will provide all but logging integration. For the logging part, Openshift Logging Operator should be leveraged.
+
+For other cases we are not aware of, we will be providing a weekly [Working Group](https://docs.google.com/document/d/1fVDDn9b69mXNyfIFpuCoOHasY0pGkDyuJ6qUyPySSdI/edit#) for internal teams as well as for building a community to support it.
+
+In all cases, MSO should not conflict with any other monitoring tool installed in the cluster and each managed service can adopt the monitoring stack on their own time. The monitoring team will support them in the migration process.
+
+### Documentation Required
+
+* Project documentation on GitHub + website
+* Tutorial on how to run it on Openshift
+* Onboarding RHOBS documentation
+
+### Open Questions
+
+* If customer can use both MSO and UWM in the future, which one should they use?
+* Should we allow customer to create MSO CRs? (MonitoringStacks).
```

Comment:
Flip question: How do we prevent them from doing that :)

Reply:
If we decide to deploy a central Grafana instance, it would be a bit weird to allow end users to deploy a monitoring stack CR. Suddenly MTSREs would see datasources from end user stacks, while end-users will not able to create dashboards for their own Prometheus instances.

Reply:
Yes. That's why I think we are leaning into the idea of enabling users to define their own, multi-ns/shared Grafana instances, outside of Monitoringstack TBH. WDYT?